src/index.ts

```diff
@@ -1 +1,18 @@
-export { default } from './Sdk';
+export { default } from './Sdk/Sdk';
```

Comment:
I think this can stay
```suggestion
export { default } from './Sdk';
```

no?